README.md

```diff
@@ -2,12 +2,13 @@
 
 Ruby refactor is inspired by the Vim plugin vim-refactoring-ruby, currently found at https://github.com/ecomba/vim-ruby-refactoring.
 
-I've implemented 5 refactorings
+I've implemented 6 refactorings
```

Comment:
At this point, it should probably read 'There are <n> refactorings available".  :)